interface-spec.md

```diff
@@ -108,11 +108,17 @@ Triple is an alias of Quad.
 
 ### DataFactory
 
+For default values of the instance properties, see the individual [interface
+definitions](#data-interfaces)
+
 **Methods:**
 
 - `IRI .iri(String iri)` returns a new instance of IRI.
 - `BlankNode .blankNode()` returns a new instance of BlankNode.
-- `Literal .literal(String value, String language, String datatype)` returns a new instance of Literal.
+- Literal .literal(String value, [String languageOrDatatype]) returns a new
+  instance of Literal. If languageOrDatatype is an IRI, then a NamedNode is
```

Comment:
@RubenVerborgh I think Nicola is asking 'how do you detect what the `languageOrDatatype` parameter is, whether it's an IRI or a language code?'.

@nicola - the idea, from what I understand, is to first check if it's a language code (which I assume is easier, from a RegEx standpoint), and if it isn't, assume that it's an IRI.